modules/ROOT/pages/proxy-deploy-raml-oas-proxy.adoc

```diff
@@ -4,21 +4,29 @@ API Manager 2.x allows you to use a RAML/OAS type of API Proxy, for all your Mul
 
 == Configuring your RAML/OAS API Proxy
 
-. In API Manager, navigate to *Manage API* to create your API Instance. +
-You can choose to either xref:manage-exchange-api-task.adoc[Creating an API instance by importing from Exchange] or xref:manage-exchange-api-task.adoc[Creating an API instance by importing on API Manager].
-. In *Asset Type*, select *RAML/OAS*.
-. To configure your API Proxy for this API instance you need to select *Endpoint with Proxy* in the *Managing Type* fields.
-.. Select your *Proxy Deployment Target*  from the available deployment targets displayed.
-.. Select the *Check this box if you are managing this API in Mule 4 or above* check mark.
-.. Type in your *Implementation URI*.
+. In API Manager, navigate to *Manage API* to create your API Instance. 
++
+You can create an API instance by importing from Exchange or by importing in API Manager. 
+. In *Asset Type*, select *REST/OAS*.
+. To configure your API Proxy for this API instance, select *Endpoint with Proxy* in the *Managing type* field.
+. In the *Proxy Deployment Target* field, select your deployment target for the proxy.
+. Select the *Check this box if you are managing this API in Mule 4 or above* checkbox.
++
+When you check the option, the strict validation options are automatically displayed under the *Validations* field.
```

Comment:
```suggestion
The strict validation options are automatically displayed under the *Validations* field.
```
...BUT do we really have to document the result of the step? Don't we assume that the step worked, and that the user can see the result in the UI?

Reply:
Fixed! Thanks!